compiler/damlc/tests/daml-test-files/Collision.daml

```diff
@@ -1,4 +1,4 @@
--- @ERROR collision between record Collision:RECORD and record Collision:Record
```

Comment:
Where is the code that produced the previous error? Can we kill that?

Reply:
It's in the Scala collision checker :) I think it's best we keep it.

Reply:
Deleted Scala code is the best Scala code :trollface:

Reply:
Delete code is the best code.

Reply:
🔥